### PR TITLE
[KNI] bump kube deps: 1.26.3 -> 1.26.7

### PR DESCRIPTION
**Pull request by swatisehgal**
Kubernetes dependency bump

 Signed-off-by: Swati Sehgal <swsehgal@redhat.com>

### CODE REVIEW

**Comment by ffromani** [comment]
it seems we need a counterpart of https://github.com/openshift-kni/numaresources-operator/pull/664 on this repo

File: build/noderesourcetopology-plugin/Dockerfile.tools [hunk 1]
```diff
@@ -17,7 +17,6 @@ RUN yum --setopt=install_weak_deps=False -y install \
     gettext \
     which \
     findutils \
-    python2 \
```
Comment (ffromani) [inline build/noderesourcetopology-plugin/Dockerfile.tools]: better to move this commit to its own PR (which will still hit broken CI) and force-merge it. Then this PR should pass CI

Comment (swatisehgal) [reply build/noderesourcetopology-plugin/Dockerfile.tools]: Ack, Thanks, will do.

**Comment by ffromani** [comment]
#117 merged! please rebase on top of the main branch, CI should pass now

**Pull request merged**